packages/core/src/utils.js

```diff
@@ -18,6 +19,14 @@ export function hostname(url) {
   return new URL(url).hostname;
 }
 
+// Return mime-type from a URL.
+export function detectMimeType(link) {
+  // mime package does not handle query params
+  let urlObj = new URL(link);
+  let urlWithoutSearchParams = urlObj.origin + urlObj.pathname;
+  return mime.lookup(urlWithoutSearchParams);
```

Comment:
This doesn't fail correct?, if it fails what happens?

Reply:
This is existing logic which is used in network.js for each resource so it shouldn't fail.